packages/protocol/contracts/bridge/EtherVault.sol

```diff
@@ -65,14 +65,30 @@ contract EtherVault is EssentialContract {
      *********************/
 
     /**
-     * Send Ether from EtherVault to the sender, checking they are authorized.
+     * Transfer Ether from EtherVault to the sender, checking that the sender
+     * is authorized.
      * @param amount Amount of ether to send.
      */
-    function receiveEther(uint256 amount) public onlyAuthorized nonReentrant {
+    function returnEther(uint256 amount) public onlyAuthorized nonReentrant {
         msg.sender.sendEther(amount);
         emit EtherTransferred(msg.sender, amount);
     }
 
+    /**
+     * Transfer Ether from EtherVault to an desinated address, checking that the
+     * sender is authorized.
+     * @param recipient Address to receive Ether
+     * @param amount Amount of ether to send.
+     */
+    function returnEtherTo(
```

Comment:
same thing, `transferEtherTo` ?

Reply:
Do you prefer "releaseEtherTo" over "returnEtherTo"?